plugins/ssr/ssrparser.mlg

```diff
@@ -2650,6 +2650,11 @@ TACTIC EXTEND ssrgenhave2
     V82.tactic (wlogtac ist pats fwd hint false (`Gen id)) }
 END
 
+TACTIC EXTEND under
+  | [ "under" ne_ident_list(names) ":" ssrrwarg(r) ssrhint(hint) ] ->
```

Comment:
TODO @gares: 
```
  under {occ}[pat]ssrterm =>ipat do [t1 | tn ]
```
then extract from `ipat` the first names of the first branch. If the name is `_` or `?` or ... return `None` (list of `Id.t option`).

Reply:
I've discussed with @ggonthier regarding our question about `move=> *` and he agreed that it could be quite useful to make `under eq_bigr do …` a shortcut for `under eq_bigr => * do …`.

Let's recap the advantages:
* we'll never want that `under eq_bigr do …` be a no-op regarding the intros, because there will be at least one binder somewhere to introduce;
* if the desired behavior is `move=> *` on all goals, we earn 5 characters (` => *`), which can be useful in one-liner mode with several nested bigops or so;
* if we have several subgoals and we don't want to do `=> *` everywhere, we can always write `=>[i| ]`.